src/stactools/landsat/utils.py

```diff
@@ -83,7 +83,10 @@ def handle_antimeridian(item: Item, antimeridian_strategy: Strategy) -> Item:
         Item: The original PySTAC Item, with updated antimeridian geometry.
     """
     geometry = shape(item.geometry)
-    if isinstance(geometry, MultiPolygon):
+    if (
+        isinstance(geometry, MultiPolygon)
+        and antimeridian_strategy == Strategy.NORMALIZE
+    ):
```

Comment:
I checked with @pjhartzell that the pre-split USGS geometries are just fine, so we only need to do this hack if we're normalizing, which is deprecated anyways.